errors/import-next.md

```diff
@@ -0,0 +1,17 @@
+# Invalid "next" Import
+
+#### Why This Error Occurred
+
+Somewhere in your application, you imported `next` directly which is only meant to be used with legacy custom servers.
+
+You should not need to import `next` inside of pages or components.
+
+#### Possible Ways to Fix It
+
+Ensure any usage of `import next from "next"` is specific to custom server usage and isn't included in your pages or components.
+
+Also ensure any type imports are kept inside of TypeScript files.
```

Comment:
Should we provide an example here. Not sure what this means?

Reply:
Added an example here https://github.com/vercel/next.js/pull/35884/commits/e697f1225395043a8484e62c43fee6b0e2e28f84